src/libraries/System.Runtime.InteropServices/gen/ComInterfaceGenerator/GeneratorDiagnostics.cs

```diff
@@ -145,6 +145,16 @@ public class Ids
                 isEnabledByDefault: true,
                 description: GetResourceString(nameof(SR.ConfigurationNotSupportedDescription)));
 
+        public static readonly DiagnosticDescriptor InterfaceTypeNotSupported =
```

Comment:
Do we want this here or in a separate file specifically for analyzer diagnostics (as opposed to generator diagnostics)? We kept them separate for `LibraryImportGenerator`. I could see keeping the `DiagnosticDescriptor` separate, but merging the IDs into one file so that it is easy to see them all / which values are used in one place. Either way, we should make both the interop generators follow the same pattern.

Reply:
I've moved them to their own file in the Analyzers/ folder.